bookshelf/php.ini

```diff
@@ -0,0 +1 @@
+suhosin.executor.func.blacklist="escapeshellarg, escapeshellcmd, exec, highlight_file, lchgrp, lchown, link, symlink, passthru, pclose, popen, proc_close, prog_get_status, proc_nice, proc_open, proc_terminate, shell_exec, show_source, system, gc_collect_cycles, gc_enable, gc_disable, gc_enabled, getmypid, getmyuid, getmygid, getrusage, getmyinode, get_current_user, libxml_disable_entity_loader, parse_str, phpinfo, phpversion"
```

Comment:
It's used by google-auth-library-php:
https://github.com/google/google-auth-library-php/blob/213ca8725ec3e3c5f5ee6560fa9f277b1cde0994/src/CredentialsLoader.php#L48